flixel/input/keyboard/FlxKey.hx

```diff
@@ -63,6 +63,8 @@ abstract FlxKey(Int) from Int to Int
 	var RBRACKET = 221;
 	var BACKSLASH = 220;
 	var CAPSLOCK = 20;
+	var SCROLL_LOCK = 145;
```

Comment:
I tried pressing scroll lock on my On-Screen Keyboard, and `FlxG.keys.pressed.SCROLL_LOCK` never returned true. have you tried this on your machine?